### PR TITLE
Add tooltips to icon buttons, add snackbar on copy button click

### LINKED ISSUE #24: Add tooltips to icon buttons
# Summary
Across whole application, icon buttons should have a tooltip describing button's behavior.

# Proposal
Wrap expansion list buttons with MudBlazor tooltips.

### PR DESCRIPTION
**Pull request by rzerdik**
fixes #24 

added tooltips to icon buttons, added snackbar after clicking on copy to clipboard in report dialog

### CODE REVIEW

File: src/Client/Dialogs/ReportDialog.razor.cs
Comment (PlesnikJakub) [inline src/Client/Dialogs/ReportDialog.razor.cs]: _"The exception that is thrown when a null reference (Nothing in Visual Basic) is passed to a method that does not accept it as a valid argument."_

So basically I am thinking if it's valid to throw ArgumentNullException when Snackbar is not an argument of a given method. 🤔 Personally I think InvalidOperationException is right choice. 

_The exception that is thrown when a method call is invalid for the object's current state._

File: src/Client/wwwroot/css/app.css
Comment (PlesnikJakub) [inline src/Client/wwwroot/css/app.css:79]: I guess there should be a new line but thats minor issue 🙈

**Review by PlesnikJakub — CHANGES_REQUESTED** [review]

File: src/Client/Dialogs/ReportDialog.razor.cs
Comment (rzerdik) [reply src/Client/Dialogs/ReportDialog.razor.cs]: You are definitely right on this one, I've changed it within the latest commit. ☺

File: src/Client/Pages/Index.razor
Comment (fpindej) [inline src/Client/Pages/Index.razor:77]: You really hate that Index component 😆

**Review by fpindej — APPROVED** [review]

File: src/Client/Pages/Index.razor
Comment (rzerdik) [reply src/Client/Pages/Index.razor:77]: I hate it more than anything else 🤕

**Review by fpindej — APPROVED** [review]
I re-requested review on accident, it should be fine. Let's just wait for @PlesnikJakub approval with his requested changes 😊

File: src/Client/Dialogs/ReportDialog.razor.cs
Comment (petrspelos) [inline src/Client/Dialogs/ReportDialog.razor.cs:66]: Defaulting to `string.Empty` instead of a `null` string parameter, should probably not be a responsibility of the caller.

But that's a minor API change. Nothing to block this from merging. ☺️

Comment (petrspelos) [reply src/Client/Dialogs/ReportDialog.razor.cs:66]: Also, rather than asserting `ClipboardService` is not null here, you'd be better off checking it or safely calling it as:
```cs
await ClipboardService?.WriteTextAsync(ReportText ?? string.Empty);
```

But again, that's just a minor point.

**Review by petrspelos — APPROVED** [review]
Seems ready for me. ☺️ Nice one, @thezerda. ❤️

It's shaping up to be pretty feature-full piece of software.

As for my suggestions, feel free to implement them if you feel it necessary, otherwise, we can have some issues created for them and mark them with the `first-timers-only` label to get some completely new people fixing those.

After all, those are usually one-line changes. 🙌

**Review by PlesnikJakub — APPROVED** [review]
:shipit:

**Pull request merged**
